kv/kv.go

```diff
@@ -238,12 +238,20 @@ type Transaction interface {
 // Client is used to send request to KV layer.
 type Client interface {
```

Comment:
does br or lighting use this interface? I am not sure if there are other code replying on this interface, do you carefully checked them all?

Reply:
As `BR` and `lightning` are merged in tidb repo now, I think it's ok.